debian9/BUILD

```diff
@@ -0,0 +1,81 @@
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@base_images_docker//util:run.bzl", "container_run_and_extract")
+load(
+    "@bazel_toolchains//rules/container:docker_toolchains.bzl",
+    "toolchain_container",
+)
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+toolchain_container(
+    name = "builder",
+    base = "@debian_base//image",
+    files = [":mkimage.sh"],
+    packages = [
+        "debootstrap",
+    ],
+)
+
+container_run_and_extract(
```

Comment:
add brief doc above about what this does (does this also need a TODO to push this file to GCS bucket)?

Reply:
Done. I've pushed it to the GCS bucket.